<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.2">
    <title>Links for abusive-error-messages</title>
  </head>
  <body>
    <h1>Links for abusive-error-messages</h1>
<a href="https://files.pythonhosted.org/packages/cd/7c/8025cdf4c5f7db558f3c8e616eeed7ae6dc034e8634f7d2ea5c4e1f7d613/abusive_error_messages-1.0.0-py3-none-any.whl#sha256=4c6e56b8fa80dbdb7ba40b438b39851cd269d8a5da49bf67089a68eac4372f44" data-requires-python="&gt;=3.8" data-dist-info-metadata="sha256=5c4823e51e159b5d9dd2b1596464a0f63100e7607bf91a602e7caac1e96f2b4f" data-core-metadata="sha256=5c4823e51e159b5d9dd2b1596464a0f63100e7607bf91a602e7caac1e96f2b4f">abusive_error_messages-1.0.0-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/a3/49/994672fec796e7e52df56696e0b9612d61bb4d0c151d324b505f35e3b525/abusive_error_messages-1.0.0.tar.gz#sha256=19cef59a9b9371b34ebfb57f085e96cfa804b3e98a37cc961b868e502c9b71ce" data-requires-python="&gt;=3.8" >abusive_error_messages-1.0.0.tar.gz</a><br />
<a href="https://files.pythonhosted.org/packages/c5/b5/fce44a2dbbcb00b1a632b9f908758736e8ba7ec4732579de914e69d272e5/abusive_error_messages-1.0.1-py3-none-any.whl#sha256=262a35399f2467776beb11174e76b8c1ba62e97e20c41fd7010b0344db27cdbc" data-requires-python="&gt;=3.8" data-dist-info-metadata="sha256=4c9391c19a76ea4b740d9e54cadac1205b547238ac6b00b1c4401c06ba18dd05" data-core-metadata="sha256=4c9391c19a76ea4b740d9e54cadac1205b547238ac6b00b1c4401c06ba18dd05">abusive_error_messages-1.0.1-py3-none-any.whl</a><br />
<a href="https://files.pythonhosted.org/packages/af/b7/3de657714435c1295ca8f219dd5292c60c2a16f269a12888b9b7a45a5ede/abusive_error_messages-1.0.1.tar.gz#sha256=988f8397267e8a7e131d8432a0c334193315e43e4307198e1039ffbae2a396df" data-requires-python="&gt;=3.8" >abusive_error_messages-1.0.1.tar.gz</a><br />
</body>
</html>
<!--SERIAL 22609640-->